WindowsAppSDK.Build.Cpp.props

```diff
@@ -71,11 +71,11 @@
     </ClCompile>
     <Link>
       <!-- Link-time optimizations:
-           /LCTG:incremental Enable Fast Link Time Code Generation
+           /LCTG Enable Link Time Code Generation
            /OPT:ICF Enable COMDAT folding
            /OPT:REF Optimize references
         -->
-      <LinkTimeCodeGeneration>UseFastLinkTimeCodeGeneration</LinkTimeCodeGeneration>
+      <LinkTimeCodeGeneration>UseLinkTimeCodeGeneration</LinkTimeCodeGeneration>
```

Comment:
Why?

Reply:
Guidance I'd gotten was /LTCG. I'm not familiar with the details between them. Does FastLTCG produce the same results in less time? Less optimization?

Reply:
Is my understanding that UseFastLinkTimeCodeGeneration equates to [`/LTCG:Incremental`](https://docs.microsoft.com/cpp/build/reference/ltcg-link-time-code-generation), and that this change will make inner loop dev miserable. (Untested caveat goes here.)

Reply:
Incremental linking is bad for performance on release builds.  It seems like a fine thing to do for debug builds.  This whole block is already scoped to release builds so disabling incremental LTCG seems reasonable to me.

Reply:
That's Foundation's model - folks generally use Debug as it's debuggable (without trying to follow the crazy instruction pointer). Occasionally when someone needs Release (benchmarking etc) they bite the bullet, and the project's not that big that it's painful.

Plus devs can always locally hack settings for local use (just don't checkin).